datasets/kor_3i4k/kor_3i4k.py

```diff
@@ -0,0 +1,94 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""3i4K: Intonation-aided intention identification for Korean dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+_CITATION = """\
+@article{cho2018speech,
+    title={Speech Intention Understanding in a Head-final Language: A Disambiguation Utilizing Intonation-dependency},
+    author={Cho, Won Ik and Lee, Hyeon Seung and Yoon, Ji Won and Kim, Seok Min and Kim, Nam Soo},
+    journal={arXiv preprint arXiv:1811.04231},
+    year={2018}
+}
+"""
+
+_DESCRIPTION = """\
+This dataset is designed to identify speaker intention based on real-life spoken utterance in Korean into one of
+7 categories: fragment, description, question, command, rhetorical question, rhetorical command, utterances.
+"""
+
+_HOMEPAGE = "https://github.com/warnikchow/3i4k"
+
+_LICENSE = "CC BY-SA-4.0"
+
+_TRAIN_DOWNLOAD_URL = "https://raw.githubusercontent.com/warnikchow/3i4k/master/data/train_val_test/fci_train_val.txt"
+_TEST_DOWNLOAD_URL = "https://raw.githubusercontent.com/warnikchow/3i4k/master/data/train_val_test/fci_test.txt"
+
+
+class Kor3i4K(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Kor_3i4k(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `kor_3i4k`